app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardFragment.kt

```diff
@@ -220,18 +220,11 @@ class DashboardFragment : BaseFragment(R.layout.fragment_dashboard) {
                 controller.someoneHasRecoveredHealthStatus = it
             }
 
-        disposables += viewModel.observeExposureNotificationPhase()
+        disposables += viewModel.observeExposureSDKReadyToStart()
             .observeOnMainThread()
-            .subscribe { phase ->
-                controller.exposureNotificationPhase = phase
-                when (phase) {
-                    is FrameworkError.Critical.ResolutionRequired -> {
-                        phase.exception.status.startResolutionForResult(
-                            requireActivity(),
-                            REQUEST_CODE_EXPOSURE_NOTIFICATION_RESOLUTION_REQUIRED
-                        )
-                    }
-                    is FrameworkError.Critical.Unknown -> handleBaseCoronaErrors(phase.exception)
+            .subscribe {
+                if (it) {
```

Comment:
For better readability, we can rename `it` to `readyToStart`

Reply:
done